### ISSUE TITLE
the bot toss the item after crafting

### ISSUE THREAD

**Issue by rom1504** [PrismarineJS/mineflayer#142]
it wasn't doing that before, but now it happens every time
just craft anything, the bot is going to toss it once it's done
if you are lucky he is going to toss it close of him so he'll get the item anyway but if you are not lucky he won't get it.

**Comment by duralakun** [comment]
I just tested it several times on a vanilla server (v 1.12) and the bot always throws the crafted items away. I was not able to make it so it doesn't throw them away.
I tested it by telling the bot "craft 1 planks" while the bot has 1 log in inventory (nothing else) (and no access to a crafting table) 

I tried to implement my own crafting method with the help of the clickWindow function.
But this also throws away the items as soon as the bot clicks slot 0 (where the crafted item is).

So i think there is some bug when using the clickWindow function on slot 0
